CONTRIBUTING.md

```diff
@@ -25,12 +25,12 @@ accept your pull requests.
 1. The repo owner will respond to your issue promptly.
 1. If your proposed change is accepted, and you haven't already done so, sign a
    Contributor License Agreement (see details above).
-1. Fork the desired repo, develop and test your code changes.
+1. Fork the desired repo, develop and test your code changes. (For technical guidance on how to fork a repo on Github, see https://help.github.com/en/articles/fork-a-repo)
```

Comment:
Maybe this:

For technical guidance on how to fork a repo on Github, see [this article] (https://help.github.com/en/articles/fork-a-repo)

Remove space in code above between closing square bracket and opening round bracket to see desired effect.